projects/packages/videopress/src/client/block-editor/blocks/video/components/details-panel/learn-how-notice/index.tsx

```diff
@@ -0,0 +1,90 @@
+/**
```

Comment:
Should we rename de folder to keep consistency with the name of the component?

`LearnHowModal` -> `learn-how-modal`